inductiva/fluids/post_processing/openfoam.py

```diff
@@ -37,31 +39,56 @@ def __init__(self, sim_output_path: types.Path):
 
         Args:
             sim_output_path: Path to simulation output files.
-        
-        Attributes:
-            sim_output_path: Path to simulation output files.
-            last_iteration: Last iteration of the simulation.
-                Obtained through the output folders.
+            default_output_files_list: List of default output files
+                that are computed on the backend and saved to the
+                simulation output directory.
         """
 
         self.sim_output_path = sim_output_path
-        # Sort all output folders and files by alphabetical order.
-        outputs_dir_list = sorted(os.listdir(sim_output_path))
-        # The second folder is the folder containing the output of
-        # the last iteration.
-        self.last_iteration = float(outputs_dir_list[1])
+        self.full_output = self.inspect_output()
+
+    def inspect_output(self):
+        """Inspect the output of the simulation."""
+
+        # TODO: Transport these files in a simpler way from the task.
+        # For such, a change to all of the post-processing output classes may
+        # be required.
+        default_output_files_list = [
+            "pressure_field.vtk", "streamlines.vtk", "stdout.txt", "stderr.txt",
+            "force_coefficients.csv", "xy_flow_slice.vtk", "xz_flow_slice.vtk",
+            "yz_flow_slice.vtk", "constant/triSurface/object.obj"
+        ]
+
+        sim_output_files = glob.glob(os.path.join(self.sim_output_path, "**"),
+                                     recursive=True)
+
+        if sim_output_files == default_output_files_list:
+            return False
+
+        return True
 
     def get_output_mesh(self):  # pylint: disable=unused-argument
```

Comment:
can't you erase now the pylint warning? you are using the self argument inside the method.

Reply:
Yes!